src/sync/client.rs

```diff
@@ -117,6 +117,15 @@ impl Client {
                     Err(x) => match x {
                         Error::Socket(y) => {
                             trace!("Socket error {}", y);
+                            let mut map = recver_map_orig.lock().unwrap();
+                            for (_, recever_tx) in map.iter_mut() {
```

Comment:
I think the variable recever_tx should be consistent with before.

Reply:
use recver_tx or correct the name all to receiver_tx

Reply:
done